### DISCUSSION TITLE
Armor Stand

### DISCUSSION THREAD

**Discussion by Dinocraft1515YT** [Draylar/identity#505] [Q&A] [answered]
How can we get the armor stand transformation in survival?

**Comment by Dinocraft1515YT** [comment]
I'm in 1.19.2 forge and I can't get it

**Reply by James103** [reply, marked as answer]
The above command errors with "Unknown entity 'minecraft:armorstand'".
The correct command is: `identity grant @s minecraft:armor_stand`

**Reply by Dinocraft1515YT** [reply]
And without commands?
